content/efcore/getting-started/samples/manipulating-entities/adding-related-entities/Program.cs

```diff
@@ -28,9 +29,11 @@ public static void Main()
             var addedAuthor = context.Authors
                 .Include(a => a.Books)
                 .Single(a => a.LastName.Contains("Shelley"));
+				
+			Console.WriteLine("We added the author {0} {1} with their book(s) -", addedAuthor.FirstName, addedAuthor.LastName);
 
             foreach (Book book in addedAuthor.Books) {
-                Console.WriteLine("{0} - {1}, {2}", book.Title, book.Genre, book.PublicationYear);
+                Console.WriteLine("\nTitle: {0}\n Genre: {1}\n Publication Year: {2}", book.Title, book.Genre, book.PublicationYear);
```

Comment:
Use multiple `Console.WriteLine` statements instead of the newlines. It will make the code more readable.